bundles/automation/org.eclipse.smarthome.automation.module.core/ESH-INF/automation/moduletypes/EventTriggersTypeDefinition.json

```diff
@@ -110,6 +110,41 @@
                "reference":"itemStateChangeTriggerID.event"
             }
          ]
+      },
+      {
+         "uid":"ThingEventTrigger",
+         "label":"Thing Trigger Event Trigger",
+         "description":"This triggers a rule if a channel emits a trigger",
+         "configDescriptions":[
+            {
+               "name":"channelId",
+               "type":"TEXT",
+               "context":"channel",
+               "label":"channel ID",
+               "description":"the id of the channel which should be observed for triggers",
+               "required":true
+            }
+         ],
+         "children":[
+            {
+               "id":"thingTriggerEventId",
+               "type":"GenericEventTrigger",
+               "configuration":{
+                  "eventSource":"$channelId",
+                  "eventTopic":"smarthome/channels/*/triggered",
+                  "eventTypes":"ThingTriggerEvent"
+               }
+            }
+         ],
+         "outputs":[
+            {
+               "name":"event",
```

Comment:
I'd be fine with that, because the topic etc. is "raw" information that is not really the level of this module anymore.